gammapy/cube/psf_map.py

```diff
@@ -229,3 +264,34 @@ def containment_radius_map(self, energy, fraction=0.68):
             m.fill_by_coord(coord, containment_radius)
 
         return m
+
+    def stack(self, other):
+        """Stack PSFMap with another one.
+
+        This works only if the PSFMap to be stacked contain compatible exposure maps.
+
+        Parameters
+        ----------
+        other : `~gammapy.cube.PSFMap`
+            the psfmap to be stacked with this one.
+
+        Returns
+        -------
+        new : `~gammapy.cube.PSFMap`
+            the stacked psfmap
+        """
+        if self.exposure_map is None or other.exposure_map is None:
+            raise (ValueError, "Missing exposure map for PSFMap.stack")
+
+        total_exposure = self.exposure_map + other.exposure_map
+        stacked_psf_quantity = (
+            self.quantity * self.exposure_map.quantity[:, np.newaxis, :, :]
```

Comment:
Suggest to put the `self.exposure_map.quantity[:, np.newaxis, :, :]` on a separate line and assign to a local variable, to avoid this line and the next ones wrapping.